### DISCUSSION TITLE
Genuinely help please

### DISCUSSION THREAD

**Discussion by Squidbeak431** [3kh0/ext-remover#544] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Ok so I've tried the first ext remover and it doesn't work and I've tried ingot and the button won't go into my bookmarks????? Wtf is going on dude

**Comment by Aka-but-nice** [comment, marked as answer]
> Ok so I've tried the first ext remover and it doesn't work and I've tried ingot and the button won't go into my bookmarks????? Wtf is going on dude

Well, which blocker do you have?
